awx/main/isolated/manager.py

```diff
@@ -153,6 +169,12 @@ def dispatch(self, playbook=None, module=None, module_args=None):
         runner_obj = self.run_management_playbook('run_isolated.yml',
                                                   self.private_data_dir,
                                                   extravars=extravars)
+
+        # Do we want to do this for all jobs rather that just containerized jobs?
+        if runner_obj.status == 'failed' and self.instance.is_containerized:
```

Comment:
This is a good idea, and I think we should do it regardless of whether it's containerized or not.